config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -619,14 +619,14 @@ presubmits:
   - name: pull-cert-manager-e2e-v1-18
     cluster: gke
     context: pull-cert-manager-e2e-v1-18
-    always_run: false
-    optional: true
+    always_run: true
+    optional: false
```

Comment:
You need to set `optional: true` and `always_run: false` on the v1.17 job too :)